README.md

```diff
@@ -0,0 +1,34 @@
+# Ethereum APIs
+
+This repository hosts service interface definitions of Ethereum (2.0) APIs.
+These definitions support both REST and gRPC protocols. 
+
+## Versioning Schedule
+
+TODO
+
+## Building / Generating Client Libraries
+
+TODO
+
+## RESTful Endpoints (gRPC Gateway)
+
+All of the gRPC services define JSON over HTTP endpoints. Client implementations
+may choose to bundle a REST service of gRPC with their client binaries or
+they may use a JSON encoding proxy such as [Envoy Proxy](https://www.envoyproxy.io/),
+[grpc-gateway](https://github.com/grpc-ecosystem/grpc-gateway), or others.
+
+## JSON Mapping
+
+The majority of field types for Ethereum are `bytes` or `uint64` type. The JSON
```

Comment:
Some background here on why this important could be useful to the reader. Namely, avoiding encoding bytes as hex string representations and issues that can cause in languages such as Go. Having a standard of base64 encoding keeps everything agnostic between languages and easy to reason about.

Reply:
Good thinking. Base64 encoding is also more space efficient. I'll add some more details about binary data representation since it is helpful for Ethereum 1 transition.